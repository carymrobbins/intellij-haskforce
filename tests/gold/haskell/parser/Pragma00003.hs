module Pragma00003 {-# DEPRECATED "Use Wobble instead" #-} where

