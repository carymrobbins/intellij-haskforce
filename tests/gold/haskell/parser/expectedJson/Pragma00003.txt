[{"Module":[{"srcInfoSpan":{"srcSpanFilename":"A.hs","srcSpanEndColumn":1,"srcSpanStartLine":1,"srcSpanEndLine":2,"srcSpanStartColumn":1},"srcInfoPoints":[{"srcSpanFilename":"A.hs","srcSpanEndColumn":1,"srcSpanStartLine":1,"srcSpanEndLine":1,"srcSpanStartColumn":1},{"srcSpanFilename":"A.hs","srcSpanEndColumn":1,"srcSpanStartLine":1,"srcSpanEndLine":1,"srcSpanStartColumn":1},{"srcSpanFilename":"A.hs","srcSpanEndColumn":1,"srcSpanStartLine":2,"srcSpanEndLine":2,"srcSpanStartColumn":1},{"srcSpanFilename":"A.hs","srcSpanEndColumn":1,"srcSpanStartLine":2,"srcSpanEndLine":2,"srcSpanStartColumn":1}]},{"ModuleHead":[{"srcInfoSpan":{"srcSpanFilename":"A.hs","srcSpanEndColumn":65,"srcSpanStartLine":1,"srcSpanEndLine":1,"srcSpanStartColumn":1},"srcInfoPoints":[{"srcSpanFilename":"A.hs","srcSpanEndColumn":7,"srcSpanStartLine":1,"srcSpanEndLine":1,"srcSpanStartColumn":1},{"srcSpanFilename":"A.hs","srcSpanEndColumn":65,"srcSpanStartLine":1,"srcSpanEndLine":1,"srcSpanStartColumn":60}]},{"ModuleName":[{"srcInfoSpan":{"srcSpanFilename":"A.hs","srcSpanEndColumn":19,"srcSpanStartLine":1,"srcSpanEndLine":1,"srcSpanStartColumn":8},"srcInfoPoints":[]},"Pragma00003"]},{"DeprText":[{"srcInfoSpan":{"srcSpanFilename":"A.hs","srcSpanEndColumn":59,"srcSpanStartLine":1,"srcSpanEndLine":1,"srcSpanStartColumn":20},"srcInfoPoints":[{"srcSpanFilename":"A.hs","srcSpanEndColumn":34,"srcSpanStartLine":1,"srcSpanEndLine":1,"srcSpanStartColumn":20},{"srcSpanFilename":"A.hs","srcSpanEndColumn":55,"srcSpanStartLine":1,"srcSpanEndLine":1,"srcSpanStartColumn":35},{"srcSpanFilename":"A.hs","srcSpanEndColumn":59,"srcSpanStartLine":1,"srcSpanEndLine":1,"srcSpanStartColumn":56}]},"Use Wobble instead"]},null]},[],[],[]]},[]]