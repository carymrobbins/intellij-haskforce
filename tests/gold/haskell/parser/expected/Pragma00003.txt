Haskell File
  MODULEDECL
    PsiElement(HaskellTokenType.module)('module')
    QCONID
      CONID
        PsiElement(HaskellTokenType.conidRegexp)('Pragma00003')
    PPRAGMA
      PsiElement(HaskellTokenType.{-#)('{-#')
      PsiElement(HaskellTokenType.PRAGMA)('DEPRECATED')
      PsiElement(HaskellTokenType.PRAGMA)('"Use')
      PsiElement(HaskellTokenType.PRAGMA)('Wobble')
      PsiElement(HaskellTokenType.PRAGMA)('instead"')
      PsiElement(HaskellTokenType.#-})('#-}')
    PsiElement(HaskellTokenType.where)('where')
  BODY
    <empty list>